src/shared/components/Header/Navigation/styles.ts

```diff
@@ -23,10 +24,17 @@ export const Navigation = styled.nav`
       color: ${Colors.BLACK};
       text-decoration: none;
     }
-    svg {
-      margin-right: 10px;
+  }
+  ${media.tabletLMax} {
+    ul {
+      display: none;
     }
```

Comment:
is there should be `tabletMax` or `tabletLMax`?

Reply:
tabletLMax